src/StatusQ/Components/StatusExpandableSettingsItem.qml

```diff
@@ -0,0 +1,150 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Rectangle {
+    id: root
+
+    property alias icon: icon
```

Comment:
As you can see in the usage of this component, because it's aliasing a `StatusIcon`, the effect of this is that we endup with something like:

```
Cmp {
  icon.icon:
}
```

This gets especially hairy the more component are being nested.

To solve this, we actually have `StatusIconSettings` and `StatusBackgroundIconSettings` which expose a similar shape, but instead of having `icon.icon` we'd end up with `icon.name` etc.

Could you update this please?

Reply:
aah this really makes sense. I didnt know about this. Will update.